pt-PT/Universally Preferable Behaviour/p02_ch07_02_more_challenging_tests_of_upb.md

```diff
@@ -1,59 +1,59 @@
-### "Don't Eat Fish"
+### "Não Comas Peixe"
 
-What would be the status of the moral proposition: “It is evil to eat fish”?
+Qual seria o status da proposição moral: "é mau comer peixe"?
 
-Clearly, this proposition seems to satisfy at least some of the requirements of UPB – it appears universal, independent of time and place, and relatively objective.
+Claramente, esta proposição parece satisfazer pelo menos alguns dos requisitos do UPB - aparenta ser universal, independente de tempo e lugar, e relativamente objetiva.
 
-Yet it seems hard for us to reasonably call this a truly *moral* theory – why?
+No entanto, parece razoalmente difícil para nós chamar isto de uma verdadeira teoria *moral* - porque?
```

Comment:
"chamar" has the wrong conjugation; "porque" is a question, no answer, and in the end of the phrase, followed by interrogation, should be "por quê"

Reply:
Thank you for the correction.